### PR TITLE
Fix/add fallback logo to credentials

### PR DESCRIPTION
**Pull request by maycon-mello**

### CODE REVIEW

File: src/features/credentials/CredentialsScreen.js [hunk 1]
```diff
@@ -126,15 +127,22 @@ const CredentialListItem = withErrorBoundary(({credential, onRemove}) => {
...
+              width={50}
+              height={50}
+              alt={''}
+              source={{
+                uri: credential.issuer.logo,
+              }}
+            />
+          ) : (
+            <PolkadotIcon
+              address={getDIDAddress(credential.issuer.id)}
```
Comment (mike-parkhill) [inline src/features/credentials/CredentialsScreen.js:142]: do we need to worry about an empty/undefined `id`?

File: src/features/credentials/credentials.js [hunk 2]
```diff
@@ -59,6 +59,12 @@ Credentials.getInstance().wallet = {
   remove: params => WalletRpc.remove(params),
 };
 
+export function getDIDAddress(did) {
+  assert(!!did, 'did is required');
```
Comment (mike-parkhill) [inline src/features/credentials/credentials.js:63]: ah, you're checking it here - nvm.

File: src/features/credentials/credentials.test.js [hunk 3]
```diff
@@ -64,6 +65,23 @@ describe('Credentials helpers', () => {
     });
   });
 
+  it('getDIDAddress', () => {
```
Comment (mike-parkhill) [inline src/features/credentials/credentials.test.js:68]: should test the assertion too by passing in null

Comment (maycon-mello) [reply src/features/credentials/credentials.test.js:68]: There is a test for it in the line 69

Comment (mike-parkhill) [reply src/features/credentials/credentials.test.js:68]: damn, so there is. 🤦

**Review by mike-parkhill — APPROVED** [review]

**Pull request merged**